### PR TITLE
Add some missing ES translations

### PR DESCRIPTION
**Pull request by ivanape**
Update ES translations

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/es.i18n.json [hunk 1]
```diff
@@ -2122,7 +2160,6 @@
   "preview-c-room": "Vista previa del canal público",
   "preview-c-room_description": "Permiso para ver los contenidos de un canal público antes de unirse",
   "Privacy": "Privacidad",
-  "Privacy_Policy": "Política de Privacidad",
```
Comment (gabriellsh) [inline packages/rocketchat-i18n/i18n/es.i18n.json]: Even though this message is "duplicated" (exists with another key), it shouldn't be removed, as it is still used in the app.

File: packages/rocketchat-i18n/i18n/es.i18n.json [hunk 2]
```diff
@@ -2366,7 +2403,6 @@
   "Select_a_user": "Seleccione un usuario",
   "Select_an_avatar": "Selecciona un avatar",
   "Select_an_option": "Seleccione una opcion",
-  "Select_department": "Seleccionar un departamento",
```
Comment (gabriellsh) [inline packages/rocketchat-i18n/i18n/es.i18n.json]: This one shouldn't be removed also.

File: packages/rocketchat-i18n/i18n/es.i18n.json [hunk 3]
```diff
@@ -444,6 +463,8 @@
   "Best_first_response_time": "Mejor tiempo de primera respuesta",
   "Block_User": "Bloquear Usuario",
   "Blockchain": "Blockchain",
+  "Blockstack_Auth_Description": "Descripción auth ",
```
Comment (gabriellsh) [inline packages/rocketchat-i18n/i18n/es.i18n.json]: White space in end of text

File: packages/rocketchat-i18n/i18n/es.i18n.json [hunk 4]
```diff
@@ -320,8 +321,14 @@
...
   "API_Gitlab_URL": "URL de GitLab",
-  "API_Personal_Access_Tokens_Regenerate_It": "Regenerar token",
-  "API_Shield_Types": "Tipos de Escudo",
+  "API_Personal_Access_Token_Name": "Nombre de Token de Acceso Personal ",
+  "API_Personal_Access_Tokens_To_REST_API": "Token de acceso personal a API REST",
+  "API_Personal_Access_Tokens_Remove_Modal": "¿Estás seguro que quieres eliminar este token de acceso personal?",
+  "API_Personal_Access_Token_Generated": "Token de Acceso Personal generado con éxito",
+  "API_Personal_Access_Token_Generated_Text_Token_s_UserId_s": "Por favor guarda cuidadosamente tu token porque no podrás visualizarlo después. <br/>Token: <strong>__token__</strong><br/>Tu ID de usuario: <strong>__userId__</strong>",
+  "API_Personal_Access_Tokens_Regenerate_Modal": "Si perdiste tu token, puedes volver a generarlo, pero recuerda que las aplicaciones que lo usen tendrán que actualizarlo",  
+  "API_Personal_Access_Tokens_Regenerate_It": "Regenerar token",
```
Comment (gabriellsh) [inline packages/rocketchat-i18n/i18n/es.i18n.json]: Added whitespace to the end of the line makes git think that this line was edited, when it wasn't. These kind of files are not linted automatically, but regardless, try and remove unnecessary changes.

Comment (programwar) [reply packages/rocketchat-i18n/i18n/es.i18n.json]: Thank you for your feedback @gabriellsh!

These changes are new lines added on translation file. This is what I see by comparing the versions:

image

File: packages/rocketchat-i18n/i18n/es.i18n.json (hunk 1 above)
Comment (programwar) [reply packages/rocketchat-i18n/i18n/es.i18n.json]: You are right. I will change it!

File: packages/rocketchat-i18n/i18n/es.i18n.json (hunk 2 above)
Comment (programwar) [reply packages/rocketchat-i18n/i18n/es.i18n.json]: You are right. I will change it!

File: packages/rocketchat-i18n/i18n/es.i18n.json (hunk 3 above)
Comment (programwar) [reply packages/rocketchat-i18n/i18n/es.i18n.json]: You are right. I will fix it!
